gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributesTest.java

```diff
@@ -29,14 +31,13 @@
 
   private static final byte[] HAPPY = "(✿◕ ‿◕ )ノ".getBytes(UTF_8);
 
-  @Rule
-  public final AppEngineRule appEngineRule = new AppEngineRule();
 
   private Path path;
   private Path dir;
 
   @Before
   public void before() throws Exception {
+    CloudStorageFileSystemProvider.setGCloudOptions(LocalGcsHelper.options());
```

Comment:
I removed it, but for the record I side with @jart here. These test methods will never have any user other than the test framework and so it seems preferable to lighten the writer's workload in this case. Having tight `throws` clauses just means we have to update them if we change the contents of the test or the signature of one of the tested method, and this work won't be compensated by the reader's decreased cognitive load.